actors/miner/src/lib.rs

```diff
@@ -3799,14 +3839,31 @@ fn extend_simple_qap_sector(
             ));
         }
 
-        new_sector.expiration = new_expiration;
-        // update deal weights to account for new duration
-        new_sector.deal_weight = deal_space * (new_sector.expiration - new_sector.activation);
-        new_sector.verified_deal_weight = BigInt::from(*new_verified_deal_space)
-            * (new_sector.expiration - new_sector.activation);
-    } else {
-        new_sector.expiration = new_expiration
+        new_sector.verified_deal_weight = BigInt::from(*new_verified_deal_space) * new_duration;
+
+        // We only bother updating the pledge for verified deals as it can increase power.
```

Comment:
nit: use the names of the fields being updated not the pledge since unless I'm missing something pledge is not updated during extension (at least in this PR)